public/about.html

```diff
@@ -33,43 +33,260 @@
+        <div class="usap_tayo_subcontent">
+          <h4>USAP TAYO!</h4>
+          <p>
+            Inilunsad din ang Usap Tayo!, mga online forum na layong palalimin ang diskurso at kaalaman ng
+            mga volunteer at kapwa kabataan patungkol sa iba’t ibang isyung panlipunan na maaaring tugunan 
+            ng mabuting pamamahala. Kabilang sa usapan ang citizen and youth empowerment, local governance, 
+            economics of social protection, public health, women empowerment, human rights, media and
+            journalism, at labor rights. Maaaring pakinggan ang mga session sa Spotify at Anchor!
+          </p>
+        </div>
+        <div class="usap_tayo_logo">
+          <img src="../assets/images/usap_tayo_logo.png">
+        </div>
+      </div>
+      <iframe src="https://open.spotify.com/embed-podcast/show/5F2Yxmz0AmVONr1h34rvBV" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
+    </div>
+  </div>
+      
+  <div class="partner_details">
+    <div class="partner_label">
+      <p>Ang Bantay Bayan ay hatid sa inyo ng mga sumusunod na organisasyon:</p>
+    </div>
+    <div class="partner_content">
+      <div class="partner_subcontent1">
+        <div class="partner_subcontent1_head">
+          <img src="../assets/images/good_gov_logo.png" class="abt_good_gov_logo">
+          <h4>GoodGovPH</h4>
+          <a href="https://facebook.com/BantayBayanTayo" target="_blank"><img src="../assets/images/about_fb.png" class="abt_fb_1"></a>
+          <a href="https://l.facebook.com/l.php?u=http%3A%2F%2Ftwitter.com%2FBantayBayanTayo%3Ffbclid%3DIwAR1XGVB-ntc44bMnby-eIy2M2PxsyXfxhafhCHt-gfrmQFuWewhdFv_Bzi8&h=AT2VaHz6xjRRLGQ4U1AhXaaHJ1joIIB_UgjrqD2nqNQj33cuhgoLPO34STyFcbMJFNg8pBQquCteBqRsFzSUqBegPJ18J-jDkgWkZsokoHSfS-1W8ELTeOxdE55cVKHi7qmbi2_IKpInrIvGfRfeSg" target="_blank"><img src="../assets/images/about_twitter.png" class="abt_twitter_1"></a>
+        </div>
+        <p>
+          Ang GoodGov PH ay isang kilusan ng mga kabataang nagsusulong ng mabuting pamamahala 
+          (“Good Governance”) sa Pilipinas. Gumagawa ito ng mga programang naka-angkla sa advocacy 
+          campaigns, alliance-building, governance education, community collaboration, at innovation. 
+        </p>
+        <p>
+          Ito ay organisasyong pangkabataan na non-profit at non-partisan, at nakarehistro sa National
+          Youth Commission - Youth Organization Registration Program (YORP). 
+        </p>
+      </div>
+      <div class="partner_subcontent2">
+        <div class="partner_subcontent2_head">
+          <img src="../assets/images/dsc_loyola_logo.png" class="abt_dsc_logo">
+          <h4>DSC - Loyola</h4>
+          <a href="https://www.facebook.com/dscloyola/" target="_blank"><img src="../assets/images/about_fb.png" class="abt_fb_2"></a>
+          <a href="https://twitter.com/dscloyola/" target="_blank"><img src="../assets/images/about_twitter.png" class="abt_twitter_2"></a>
```

Comment:
`rel="noopener noreferrer"` should be added to links containing target="_blank" as a precaution against reverse tabnabbing. For more information, please refer to the following article:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/